react-frontend/src/core-components/form/FormComponent.js

```diff
@@ -0,0 +1,8 @@
+import React from "react";
+import { Form } from "react-bootstrap";
+
+const Label = React.memo(Form.Label);
```

Comment:
a label is part of the Form itself no need to export it explicitly, just export Form, and instead of FormComponent export only `Form` it will be easy to use it in the code